source/agora/node/LederOutputStream.d

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+
+    Function definition and helper related to serialization
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.node.LedgerOutputStream;
+
+import agora.common.Transaction;
+
+/// Ditto
+public class LedgerOutputStream
+{
+    private ubyte[] data;
+
+    private size_t offset;
+
+    /// Ctor
+    public this () @trusted
```

Comment:
Why does this need to be `@trusted` ? In general, we should use `@trusted` as little as possible, and every usage should be carefully considered.

Reply:
For some reason, I used `@trusted`
It cannot be used in the event of a pointer change.